frappe/social/doctype/energy_point_log/energy_point_log.py

```diff
@@ -40,7 +39,12 @@ def after_insert(self):
 				'reference_user': reference_user
 			}
 
-			create_notification_log(self.user, notification_doc, self.reason)
+			frappe.enqueue(
+				'frappe.core.doctype.notification_log.notification_log.create_notification_log',
```

Comment:
Why not enqueue in 'frappe.core.doctype.notification_log.notification_log.create_notification_log' method itself.